packages/core/src/config/minimal.ts

```diff
@@ -110,7 +110,17 @@ const minimal: PluginStyleguideConfig<'built-in'> = {
   },
   arazzoRules: {
     spec: 'error',
-    'parameters-no-body-inside-in': 'off',
+    'parameters-not-in-body': 'off',
+    'source-description-type': 'off',
```

Comment:
I just noticed we named the other thing `sourceDescription-name-unique` but this one `source-description-*`. Can we make these consistent?

Reply:
Sure, updating now ....